internal/build/get_version.sh

```
@@ -0,0 +1,33 @@
+
+GO_MAJOR=$(echo "$GO_VERSION" | cut -d. -f1)
+GO_MINOR=$(echo "$GO_VERSION" | cut -d. -f2)
+GO_VER=$(( GO_MAJOR * 1000 + GO_MINOR ))
+
+X=$(echo "$1" | cut -d. -f1)
+Y=$(echo "$1" | cut -d. -f2)
+VER=$(( X * 1000 + Y))
+
+shift
+
+if [ $GO_VER -ge $VER -a $# -gt 0 ]; then
```

Shellcheck

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)